### PR TITLE
Add group message listview and business-logic for it

### PR DESCRIPTION
**Pull request by Snare295**
- add dependncy autos_size_text
- UI and business logic for group message list tile, tile that will be seen in a message list of group
- added all classes needed for it, in future will be used with parsing API from server

### CODE REVIEW

File: lib/ui/widgets/message/group/group_message_tile.dart [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+        throw ArgumentError(
+            "$this have constructed with isMe = $isMe, but $sender isn't user");
+      }
+    }
+  }
+
+  int? id;
+  String messageStr;
+  Sender sender;
+  bool isMe;
```
Comment (bLuDd1) [inline lib/ui/widgets/message/group/group_message_tile.dart]: Move this properties to the top of the class declaration.

File: lib/ui/widgets/message/group/group_message_tile.dart [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+    super.key,
+    this.id,
+    this.messageStr = "Uninitialised messageStr",
+    required this.sender,
+    required this.isMe,
+  }) {
+    if (isMe == true) {
+      if (sender.name != UserData().name) {
+        print(sender.name);
+        print(UserData().name);
```
Comment (bLuDd1) [inline lib/ui/widgets/message/group/group_message_tile.dart]: Delete unnecessary prints

File: lib/ui/widgets/message/group/group_message_tile.dart [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
 class GroupMessageTile extends StatefulWidget {
-  const GroupMessageTile({super.key});
+  GroupMessageTile({
+    super.key,
+    this.id,
+    this.messageStr = "Uninitialised messageStr",
+    required this.sender,
+    required this.isMe,
+  }) {
+    if (isMe == true) {
```
Comment (bLuDd1) [inline lib/ui/widgets/message/group/group_message_tile.dart]: Simplify if statement to 
```suggestion
    if (isMe) {
```

**Review by bLuDd1 — CHANGES_REQUESTED** [review]

File: lib/ui/widgets/message/group/group_message_tile.dart (hunk 2 above)
Comment (Snare295) [reply lib/ui/widgets/message/group/group_message_tile.dart]: Deleted

File: lib/ui/widgets/message/group/group_message_tile.dart (hunk 3 above)
Comment (Snare295) [reply lib/ui/widgets/message/group/group_message_tile.dart]: Ok

File: lib/ui/widgets/message/group/group_message_tile.dart (hunk 1 above)
Comment (Snare295) [reply lib/ui/widgets/message/group/group_message_tile.dart]: Moved, but constructor still will be before builder method

**Pull request merged**